compose-setup.sh

```diff
@@ -113,7 +127,21 @@ while getopts "fh" opt; do
   esac
 done
 
+cat <<EOM
+
+###########
+# WARNING #
+###########
+
+This deployment method is intended for testing/development purposes.
```

Comment:
I wonder if it would be a good idea to state that we will provide a production-ready version of this on the near future also in this message.